users/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls  import path
+from users.views  import SignUpView, LoginView
+
+urlpatterns = [
+    path('/member/new', SignUpView.as_view()),
+    path('/member', LoginView.as_view()),
```

Comment:
```suggestion
    path('/signup', SignUpView.as_view()),
    path('/signin', LoginView.as_view()),
```
작성해주신 url은 최종적으로 users/member,  users/member/new 와 같은 형태가 됩니다. 
왜 RESTful 하지 못한지 세션을 듣고 고민해봐주세요.
일반적인 로그인, 회원가입 기능의 엔드포인트는 위와 같이 작성합니다.

Reply:
넵 알겠습니다!